modules/prediction/container/adc_trajectory/adc_trajectory_container.cc

```diff
@@ -76,5 +93,47 @@ bool ADCTrajectoryContainer::ContainsLaneId(const std::string& lane_id) const {
          reference_line_lane_ids_.end();
 }
 
+Polygon2d ADCTrajectoryContainer::GetJunctionPolygon() {
+  int num_point = adc_trajectory_.trajectory_point_size();
+  if (num_point == 0) {
+    return Polygon2d{};
+  }
+  std::shared_ptr<const JunctionInfo> junction_info(nullptr);
+  double prev_s = (-2) * FLAGS_distance_to_adc_trajectory_thred;
+  for (int i = 0; i < num_point; ++i) {
+    double s = adc_trajectory_.trajectory_point(i).path_point().s();
+    if (i > 0 &&
+        std::abs(s - prev_s) < FLAGS_distance_to_adc_trajectory_thred) {
```

Comment:
FLAGS_distance_to_adc_trajectory_thred: rename it to something like adc_junction_search_radius